docs/source/daml/code-snippets-dev/Interfaces.daml

```diff
@@ -107,19 +107,3 @@ template NameOfTemplate
     interface instance YourInterface for NameOfTemplate where
       view = EmptyInterfaceView
 -- EMPTY_INTERFACE_INSTANCE_END
-
```

Comment:
Don't we need to change the docs as well if that snippet is referenced in the docs?

Reply:
nope, I used to think this at some point too but `docs.daml.com` points to specific `daml` repo commits, not just `main HEAD`. We **will** need to change the docs _at some point_ but that makes sense for a feature we're removing, it just doesn't have to be now.